### ISSUE TITLE
Yuzu Don't Open with a Generic Controller Conected (3ds Controller)

### ISSUE THREAD

**Issue by FjCardona** [yuzu-emu/yuzu#7345]
I'm trying to play Pokemon BD with a 3ds as a controller with the 3ds Controller and vJoy Software. Yuzu didn't open until I close de 3ds Controller app and when I try to open with yuzu running, but it crashes until I close it again. I had the following log:

[yuzu_log.txt.old.txt](https://github.com/yuzu-emu/yuzu/files/7549123/yuzu_log.txt.old.txt)

**Comment by FjCardona** [comment]
I forgot to say it. This didn't happend until the last 2 or 3 patches (now 811) . In the previous patches all worked correctly (Maybe 808 or before)

**Comment by Bobynours** [comment]
Hi ; i have the same issue here..

**Comment by german77** [comment]
can you verify that is works on 808. Upload a log or something else that could give me hints. We haven't done any changes to the controller support recently

**Comment by FjCardona** [comment]
Log attached. Also I tried now with 808 and 800 but still doesn´t open. Maybe a win11 incompatibility? 

[yuzu_log.txt.old.txt](https://github.com/yuzu-emu/yuzu/files/7566101/yuzu_log.txt.old.txt)

**Comment by german77** [comment]
Probably Yeah there is nothing here. Do you have a link to setup the 3ds? I think I got one laying around

**Comment by Bobynours** [comment]
This https://github.com/CTurt/3DSController ?
But the 3DScontroller work with others emulators for me.

**Comment by FjCardona** [comment]
Yeah it is, It's working for me as well in others emulators but yuzu still crashing

**Comment by Bobynours** [comment]
![YUZU message with 3dscontroller](https://user-images.githubusercontent.com/94647363/142756677-a139d88f-58a0-4748-9498-6f34cfbe25bc.png)

I saw that there is this error message like 1/4 seconds before yuzu close (were hard to have a screen capture !)

**Comment by Bobynours** [comment]
I saw that there is this error message like 1/4 seconds before yuzu close (were hard to have a screen capture !)

![YUZU message with 3dscontroller](https://user-images.githubusercontent.com/94647363/142756732-0f473c5a-92dc-4fa6-83f9-98df297c43cc.png)

**Comment by FjCardona** [comment]
I debugged yuzu with 3ds Controller opened and I got the following logs before and after closing it 
[Before.txt](https://github.com/yuzu-emu/yuzu/files/7584378/Before.txt)
[After.txt](https://github.com/yuzu-emu/yuzu/files/7584380/After.txt)

**Comment by benderscruffy** [comment]
is this still an issue ???
